base/shell/explorer/startctxmnu.cpp

```diff
@@ -29,15 +29,21 @@ class CStartMenuBtnCtxMenu :
     public CComObjectRootEx<CComMultiThreadModelNoCS>,
     public IContextMenu
 {
+    /* AddStartContextMenuItems uses ID_SHELL_CMD IDs directly and relies on idCmdFirst being 0. */
+    /* CTrayWindow::TrackCtxMenu must pass 0 because DeleteMenu ID_SHELL_CMD_UNDO_ACTION would   */
+    /* delete the wrong item if it used 1. m_Inner->QueryContextMenu is not aware of this game   */
+    /* so we have to reserve the entire ID_SHELL_CMD range for ourself here.                     */
```

Comment:
Better use
```c++
// multi
// line
// comment
```
or
```c
/* multi
 * line
 * comment */
```
In both cases this avoids having to put trailing `*/` with space padding.

Also, "ourself" --> "ourselves"